<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BubbleDataEditor</class>
 <widget class="QWidget" name="BubbleDataEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bubbles Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KoChart::ChartTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QToolButton" name="insertColumnBefore">
         <property name="toolTip">
          <string>Insert data column before selected column</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="edit-table-insert-column-left">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="insertRowBelow">
         <property name="toolTip">
          <string>Insert data row below selected row</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="edit-table-insert-row-under">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="insertRowAbove">
         <property name="toolTip">
          <string>Insert data row above selected row</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="edit-table-insert-row-above">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="insertColumnAfter">
         <property name="toolTip">
          <string>Insert data column after selected column</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="edit-table-insert-column-right">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="deleteSelection">
         <property name="toolTip">
          <string>Delete selection</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="edit-delete">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="dataSetView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="manualControl">
         <property name="toolTip">
          <string>Manual control of data sets</string>
         </property>
         <property name="text">
          <string>Manual</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="removeDataSet">
         <property name="toolTip">
          <string>Remove data set</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="edit-table-delete-row">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="addDataSetAfter">
         <property name="toolTip">
          <string>Add data set after selected data set</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="edit-table-insert-row-below">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="addDataSetBefore">
         <property name="toolTip">
          <string>Add data set after selected data set</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="edit-table-insert-row-above">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoChart::ChartTableView</class>
   <extends>QTableView</extends>
   <header>ChartTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
